### PR TITLE
Mason edited 4 pages

### PR DESCRIPTION
**Pull request by mpj2104**
Mason edited the following pages:

- `_policies/acknowledgements/`
- `_policies/security/`
- `blog/2023/05/16/sec-filings/`
- `blog/2023/09/18/transferring-files-from-the-yens-to-google-drive-using-rclone/`

### CODE REVIEW

File: docs/blog/posts/2023-05-16-EDGARmirror.md [hunk 1]
```diff
@@ -161,16 +167,16 @@ Syntax supports the following:
...
+WRDS [Bag of Words (BoW) interface](https://wrds-www.wharton.upenn.edu/pages/get-data/wrds-sec-analytics-suite/wrds-sec-text-analysis/wrds-bag-words/){:target="_blank"} provides WRDS SEC Analytics Suite subscribers an alternative way to perform content analysis against company SEC filings. A frequency count of all words in SEC filings and amendments from 1993 through 2022 was prepared. Users can query word counts across all filings, or against specific company filings. Total word occurrence per filing, cosine similarity, Jaccard similarity, minimum edit distance between current and previous filings, word stemming and lemmatization, English word, positive or negative word, stop word, geographic, company name, patent related (requires KtMINE subscription) word indicators etc. are also available. 
+You can read more about how words are contructed and what summary measures were applied in the [manual](https://wrds-www.wharton.upenn.edu/documents/1081/WRDS_Bag_of_Words_Manual.pdf){:target="_blank"}. 
 
 Here are some of the modules available:
 ```
```
Comment (natalya-patrikeeva) [inline docs/blog/posts/2023-05-16-EDGARmirror.md:182]: I would maybe add a Summary to say something about the whole page. It ends abruptly.

File: docs/blog/posts/2023-09-18-rclone.md [hunk 2]
```diff
@@ -1,25 +1,18 @@
 ---
 date:
   created: 2023-09-18
-pin: true
+categories:
+    - File Transfer
+    - rclone
+    - Google Drive
 ---
+# Transferring files from the Yens to Google Drive using `rclone`
```
Comment (natalya-patrikeeva) [inline docs/blog/posts/2023-09-18-rclone.md:9]: Can we shorten the title to fit into one line? Maybe something like "rclone files from Yens to Google drive"

File: docs/blog/posts/2023-09-18-rclone.md [hunk 3]
```diff
@@ -1,25 +1,18 @@
...
+<div class="last-updated">Last updated: 2024-09-30</div>
```
Comment (natalya-patrikeeva) [inline docs/blog/posts/2023-09-18-rclone.md:10]: don't need this

File: docs/blog/posts/2023-09-18-rclone.md [hunk 4]
```diff
@@ -1,25 +1,18 @@
...
-```
-$ curl https://rclone.org/install.sh | sudo bash
-```
-
-### Using `rclone` on Yen Servers
-To utilize `rclone` the yens, load the `rclone` module:
+#### Loading `rclone` on the Yens
+To utilize `rclone` on the Yens, first load the `rclone` module:
 
 ```bash
```
Comment (natalya-patrikeeva) [inline docs/blog/posts/2023-09-18-rclone.md:17]: add title, remove $ so readers can copy

File: docs/blog/posts/2023-09-18-rclone.md [hunk 5]
```diff
@@ -1,25 +1,18 @@
...
 $ ml rclone
```
Comment (natalya-patrikeeva) [inline docs/blog/posts/2023-09-18-rclone.md:18]: only show output, disable copy and add title that is the "Terminal Output"

**Review by natalya-patrikeeva — CHANGES_REQUESTED** [review]
I left comments on some code blocks - we are adding titles, removing $, disable copy if it's output etc. I didn't mark all of them though.

File: docs/blog/posts/2023-05-16-EDGARmirror.md [hunk 6]
```diff
@@ -1,37 +1,39 @@
...
+- /zfs/data/NODR/EDGAR_HTTPS/<span style="color: red;"><b>edgar/data/1050122/0001047469-03-017249</b></span>.txt
 
-- https://www.sec.gov/Archives/ + <span style="color: red;"><b>edgar/data/1050122/0001047469-03-017249</b></span> + -index.htm
+- https://www.sec.gov/Archives/<span style="color: red;"><b>edgar/data/1050122/0001047469-03-017249</b></span>-index.htm
 
-The above provides a clear way to identify specific documents, but not a way to "query" those documents. To address this, the DARC team also maintains a <a href="https://redivis.com/datasets/dq12-4q4st0kjt" target="_blank">SQL dataset on Redivis</a> that can be queried for specific filings.
+The above provides a clear way to identify specific documents, but not a way to "query" those documents. To address this, the DARC team also maintains a [SQL dataset on Redivis](https://redivis.com/datasets/dq12-4q4st0kjt){:target="_blank"} that can be queried for specific filings.
 
 This dataset contains a table, `edgar-filings`, with the following fields:
 ```
```
[the remark this answers is not in the document]
Comment (mpj2104) [reply docs/blog/posts/2023-05-16-EDGARmirror.md:39]: hmmm, i never explicitly defined a "copy button", it just shows up when the page gets rendered

File: docs/blog/.authors.yml [hunk 7]
```diff
@@ -6,6 +6,16 @@ authors:
...
\ No newline at end of file
+    url: https://github.com/mpj2104            # Author website URL
+  nrapstin:
+    name: Natalya Rapstine                                       # Author name
+    description: 🦄 Research Computing Specialist @DARC GSB Team  # Author description
+    avatar: https://github.com/natalya-patrikeeva.png            # Author avatar
+    url: https://github.com/natalya-patrikeeva                   # Author website URL
+  awn:
+    name: Amy Ng                                       # Author name
+    description: 🦄 Lead Research Analytics Scientist @DARC GSB Team  # Author description
```
Comment (natalya-patrikeeva) [inline docs/blog/.authors.yml:19]: Can Amy get a different emoji lol

Comment (natalya-patrikeeva) [reply docs/blog/.authors.yml:19]: In fact, we all should get an emoji

**Review by natalya-patrikeeva — APPROVED** [review]
Besides the emoji, looks good to me. We can pretty up rclone page later. I might have nit picky styling things

File: docs/blog/.authors.yml (hunk 7 above)
Comment (mpj2104) [reply docs/blog/.authors.yml]: oopsie, fixed now

**Pull request merged**